Add tidb-backend necessary instructions (#4939)
pingcap/docs: migrate-from-mysql-dumpling-files.md

    password = ""
    ```

> 3. Add the necessary parameters of TiDB-backend. This document uses TiDB-backend mode. Here, it can also be set to "local" or "importer" according to actual application scenarios. For details, refer to [Backend Mode](/tidb-lightning/tidb-lightning-backends.md).

Reviewer: ```suggestion
3. Add the necessary parameter for the backend. This document uses the TiDB-backend mode. Here, "backend" can also be set to "local" or "importer" according to your actual application scenario. For details, refer to [Backend Mode](/tidb-lightning/tidb-lightning-backends.md).
```

    ```
    [tikv-importer]
    backend = "tidb"
    ```

> 4. Add necessary parameters for importing TiDB cluster.

Reviewer: ```suggestion
4. Add necessary parameters for importing the TiDB cluster.
```
